### DISCUSSION TITLE
Numpy error in running tutorials

### DISCUSSION THREAD

**Discussion by lily-dove** [Parcels-code/Parcels#1593] [Q&A]
Hi all, I am new to OceanParcels and am just trying to run some of the tutorials right now. I installed the package and tried to run the example_peninsula.py tutorial, but I get the error 
> AttributeError: `np.infty` was removed in the NumPy 2.0 release. Use `np.inf` instead.

Any way to easily update OceanParcels to run happily with a newer version of Numpy?

**Comment by erikvansebille** [comment]
Thanks for reporting @lily-dove, and welcome to parcels!

This is indeed an issue with the new NumPy version, fixed by @andrew-s28 in https://github.com/OceanParcels/parcels/pull/1589. I've just [updated the conda version to v3.0.4](https://github.com/conda-forge/parcels-feedstock/pull/113#pullrequestreview-2144307010). Check https://anaconda.org/conda-forge/parcels whether it has also made it into the conda channels.

So if you upgrade your conda package (`conda update parcels>3.0.3`) all should work again soon(?)
